Shoko.Server/Models/SVR_VideoLocal_Place.cs

```diff
@@ -469,6 +470,65 @@ public void RemoveAndDeleteFileWithOpenTransaction(ISession session, HashSet<SVR
             }
         }
 
+        public class RelocateResult
+        {
+            public bool Success { get; set; }
+
+            public string RelativePath { get; set; }
+
+            public SVR_ImportFolder ImportFolder { get; set; }
+        }
```

Comment:
This entire implementation could be a tuple return - but again this is seeming to be a lot of unrelated changes.

Reply:
https://github.com/ShokoAnime/ShokoServer/blob/d170da7c7cc219671b10d430da36511aeef0db4d/Shoko.Server/Models/SVR_VideoLocal_Place.cs#L53

I saw this, so I decided to use a "return object" instead of a tuple.

Reply:
For a simple DTO I would prefer a tuple return as you can very easily destructure them out